2.cellprofiler_processing/cellprofiler_env.yml

```diff
@@ -0,0 +1,26 @@
+name: cp4
```

Comment:
Will this environment be used for multiple steps or directories? If so, it may be helpful to move this environment specification outside of this directory.

Reply:
@jenna-tomkinson -  making sure you saw this comment

Reply:
Discussed